insecure-content-warning.php

```diff
@@ -24,6 +24,53 @@
 define( 'INSECURE_CONTENT_INC', INSECURE_CONTENT_PATH . 'includes' . DIRECTORY_SEPARATOR );
 define( 'INSECURE_CONTENT_VERSION', '1.1.0' );
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @since 1.1.2
+ *
+ * @return string Minimum version required.
+ */
+function minimum_php_requirement(): string {
```

Comment:
```suggestion
function minimum_php_requirement() {
```

This is to keep the file compatible with PHP 5.6 (as it supports versions of WP prior to 6.3)

Reply:
Done

Reply:
(sorry, wrong profile)